writings-volume-2/updated/15-chapter-15.adoc

```diff
@@ -3548,3 +3548,4 @@ greatest loser of all who had a share in his life.
 
 [.signed-section-signature]
 Mary Penington
+//I didn't embed this ^^ because the whole chapter is letters, but I'm not sure if that's right. 
```

Comment:
Yep, I think this is the right call.  Both of the volumes have one entire chapter of letters (the last chapter of each book). And since they are nothing but letters, I think we should NOT embed them.